packages/api/package.json

```diff
@@ -28,6 +28,8 @@
     "@google-cloud/firestore": "^3.8.4",
     "body-parser": "^1.19.0",
     "express": "^4.17.1",
+    "stream": "0.0.2",
```

Comment:
Readable Stream is built in to Node.js core now, so you shouldn't need this module:

https://nodejs.org/api/stream.html#stream_class_stream_readable